### PR TITLE
[12.0] [MIG] hr_expense_operating_unit

### PR DESCRIPTION
**Pull request by bjeficent**
Migration to version 12.0

### CODE REVIEW

File: hr_expense_operating_unit/models/hr_expense.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
+# Copyright 2016-19 Eficent Business and IT Consulting Services S.L.
+# Copyright 2016-19 Serpent Consulting Services Pvt. Ltd.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).).
+
+from odoo.tools.translate import _
+from odoo import api, fields, models
```
Comment (nikul-serpentcs) [inline hr_expense_operating_unit/models/hr_expense.py]: Improve `from odoo import api, fields, models, _` and remove https://github.com/OCA/operating-unit/pull/184/files#diff-85ed473ab0a9e52c02ccb1eb7ea9aa0bR5

File: hr_expense_operating_unit/models/hr_expense.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+                                        self.env['res.users'].
+                                        operating_unit_default_get(self._uid))
+
+    @api.multi
+    @api.constrains('operating_unit_id', 'company_id')
+    def _check_company_operating_unit(self):
+        for rec in self:
+            if (rec.company_id and rec.operating_unit_id and rec.company_id !=
+                    rec.operating_unit_id.company_id):
+                raise ValidationError(_('Configuration error. The Company in '
```
Comment (nikul-serpentcs) [inline hr_expense_operating_unit/models/hr_expense.py:24]: IMO Improve indentation in all ValidationError, use Like

```
raise ValidationError(_('''Configuration error. The Company in the Expense and
in the Operating Unit must be the same.'''))
```

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
Improve Code

**Comment by bjeficent** [comment]
Comments attended, thanks for reviewing.

**Comment by AaronHForgeFlow** [comment]
HI @nikul-serpentcs can you please update your review?

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM :+1:

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Works as expected

File: hr_expense_operating_unit/tests/test_hr_expense_operating_unit.py [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+                'sheet_id': self.hr_expense_sheet1.id
+            })
+            self.hr_expense_sheet1.expense_line_ids.write({
+                'operating_unit_id': self.b2c.id
+            })
+
+        company_id = self.env['res.company'].create({
+            'name': 'My Company',
+            'partner_id': self.partner1.id,
+            'rml_header1': 'My Company',
```
Comment (nikul-serpentcs) [inline hr_expense_operating_unit/tests/test_hr_expense_operating_unit.py:156]: @bjeficent @AaronHForgeFlow @HviorForgeFlow No need to `rml_header1`

Comment (AaronHForgeFlow) [reply hr_expense_operating_unit/tests/test_hr_expense_operating_unit.py:156]: Thanks for the notice. Could you create a PR with the fix?

**Pull request merged**